app/models/partners/profile.rb

```diff
@@ -125,6 +125,13 @@ def split_pick_up_emails
       pick_up_email.split(/,|\s+/).compact_blank
     end
 
+    def county_list_by_region
+      # provides a county list in case insensitive alpha order, by region, then county name
+      counties = served_areas.map(&:county).uniq
+      counties.sort_by! { |county| [county.region.downcase, county.name.downcase] }  # Our current world only has the US states, so downcasing the region doesn't really matter.
+      counties.pluck(:name).join("; ")
```

Comment:
Can we move this to SQL? You can add `has_many :through` for counties to the profile, and then you can do something like

```
self.counties.order(%w(region name)).pluck(:name).join('; ')
```

Reply:
I don't think that will get me the downcase,  but I'm sure there's a way to get that in  there too.

Reply:
Made those changes.  Added a second profile to the test to make sure that there isn't crosstalk.